src/blueprint/into_schema.rs

```diff
@@ -30,53 +37,144 @@ fn to_type_ref(type_of: &Type) -> dynamic::TypeRef {
   }
 }
 
+fn get_cache_key<'a, H: Hasher>(ctx: &'a EvaluationContext<'a, ResolverContext<'a>>, mut hasher: H) -> u64 {
```

Comment:
will be great if we have some kind of tests to make sure there is no collision in cache keys for different configuration since there is much going on here.